pact/ism/merkle-tree-ism.pact

```diff
@@ -55,6 +55,16 @@
     4
   )
 
+  (defun add-validator (validator:string)
+    (with-capability (ONLY_ADMIN)
+      (with-read contract-state "default"
+        { "validators" := validators }
+        (enforce (not (contains validator validators)) "Validator already exists")
+        (update contract-state "default" { "validators": (+ validators [validator]) })

```

Comment:
Considering that we're checking for the existence of `validator` in `validators` already, we don't need to `update` here. We can just `write`.

```suggestion
        (write contract-state "default" { "validators": (+ validators [validator]) })

```